catalog-rest-service/src/main/resources/ui/src/utils/EntityVersionUtils.tsx

```diff
@@ -0,0 +1,196 @@
+import classNames from 'classnames';
+import { diffArrays, diffWordsWithSpace } from 'diff';
+import { isUndefined } from 'lodash';
+import React, { Fragment } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import ReactMarkdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw';
+import gfm from 'remark-gfm';
+import {
+  ChangeDescription,
+  FieldChange,
+} from '../generated/entity/services/databaseService';
+import { TagLabel } from '../generated/type/tagLabel';
+import { isValidJSONString } from './StringsUtils';
+
+/*eslint-disable */
```

Comment:
Why is whole eslint disabled?

Reply:
Because for Fieldchange type we do not get any specific type. also, the markdown library that we are using need some lint checks turned off.